src/caffe/solver.cpp

```diff
@@ -434,7 +434,8 @@ Dtype SGDSolver<Dtype>::GetLearningRate() {
         (Dtype(1.) + exp(-this->param_.gamma() * (Dtype(this->iter_) -
           Dtype(this->param_.stepsize())))));
   } else {
-    LOG(FATAL) << "Unknown learning rate policy: " << lr_policy;
+    rate = Dtype(0.);
```

Comment:
Addressed in https://github.com/matthiasplappert/caffe/commit/322a9de77b6d56f4dea0fa060bc02e7b2c7b97d5